exportsage50/__openerp__.py

```diff
@@ -23,106 +24,124 @@
     'name': 'Export to Sage50',
     'version': '1.0',
     "category": 'Accounting & Finance',
-    'complexity': "easy",
     'description': """
-French version below
 
-Export accounting data from OpenErp to Sage50.
-====================================
- Export accounting data from OpenErp to Sage50. The exportation generates the imp file to import in Sage50.
+Export accounting data from OpenERP to Sage50
+=============================================
```

Comment:
You should use OCA template for the description: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst